ext/cl_loader_layers.asciidoc

```diff
@@ -50,6 +54,12 @@ cl_int clInitLayer(cl_uint                 num_entries,
 ----
 typedef cl_uint cl_layer_info;
 typedef cl_uint cl_layer_api_version;
+typedef struct _cl_layer_option {
+    const char *name;
+    const char *value;
+    const char *default_value;
+    const char *description;
```

Comment:
Presumably these are `NUL` terminated strings?  It might be good to say this explicitly.

Reply:
Indeed, will add it.

Reply:
Done in https://github.com/KhronosGroup/OpenCL-Docs/pull/851/commits/39686e806dc0a1d27a5938de46846474785a30a9